cpp/unreal_plugins/SimulationController/Source/SimulationController/SimulationController.cpp

```diff
@@ -52,12 +53,11 @@ enum class FrameState
 void SimulationController::StartupModule()
 {
     SP_LOG_CURRENT_FUNCTION();
+    SP_ASSERT(FModuleManager::Get().IsModuleLoaded(Unreal::toFName("CommonModuleRules")));
```

Comment:
This change is from my branch but I don't think it's a good idea any more. The C++ code doesn't rely on `CommonModuleRules` so I don't think it's a good idea if the C++ code asserts that the module is loaded.

Reply:
Agreed. Updated this.